src/components/SaveExport/ExportDialog.jsx

```diff
@@ -102,34 +176,38 @@ export default () => {
         if (isExportDialogVisible) {
             cancel.current = false;
         }
-    }, [isExportDialogVisible]);
-
-    const end = windowEnd || options.timestamp;
-    const begin = windowBegin || end - windowDuration;
 
-    const [from, to] =
-        cursorBegin === null ? [begin, end] : [cursorBegin, cursorEnd];
+        if (cursorBegin != null) {
+            updateRadioSelected(2);
+        } else {
+            updateRadioSelected(0);
+        }
```

Comment:
This could be changed to `updateRadioSelected(cursorBegin != null ? 2 : 0)`, but I'm not sure if that is better ;)